avm/res/network/front-door/tests/e2e/waf-aligned/main.test.bicep

```diff
@@ -99,10 +99,10 @@ module testDeployment '../../../main.bicep' = [
         {
           name: 'heathProbe'
           properties: {
-            enabledState: ''
-            healthProbeMethod: ''
+            enabledState: 'Enabled'
```

Comment:
Hey @rozkurt, please not that this will fail the static tests faster than your can say 'static tests'. Not my best joke.

Anyways, please make sure you re-run the `Set-AVMModule` script on the module as this change here will cause an update to the readme the static validation will check for.